scipy/io/mmio.py

```diff
@@ -30,7 +33,7 @@ def mminfo(source):
     Parameters
     ----------
 
-    source : file
+    filename : file
```

Comment:
I think `source` is better, given this can be an open file-like object.   I commented on this below...